src/platform/Tizen/WiFiManager.h

```diff
@@ -0,0 +1,93 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFI
+#include <platform/internal/DeviceNetworkInfo.h>
+
+#include <glib.h>
+#include <wifi-manager.h>
+
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+
+class WiFiManager
+{
+public:
+    static void Init(void);
+    static void Deinit(void);
+
+    static CHIP_ERROR IsActivated(bool * isWifiActivated);
+    static CHIP_ERROR Activate(void);
+    static CHIP_ERROR Deactivate(void);
+    static CHIP_ERROR Connect(const char * ssid, const char * key);
+    static CHIP_ERROR Disconnect(const char * ssid);
+    static CHIP_ERROR RemoveAllConfigs(void);
+
+    static CHIP_ERROR GetDeviceState(wifi_manager_device_state_e * deviceState);
+    static CHIP_ERROR SetDeviceState(wifi_manager_device_state_e deviceState);
+    static CHIP_ERROR GetModuleState(wifi_manager_module_state_e * moduleState);
+    static CHIP_ERROR GetConnectionState(wifi_manager_connection_state_e * connectionState);
```

Comment:
Same comment as for SoftAP manager:

- Suggest avoiding static methods, altogether, that enforce singleton and make the class harder to test.
- Please document side effects, etc.

Reply:
Thank you for pointing out.
Updated it.